cfn/cfn.json

```diff
@@ -347,14 +186,9 @@
                     }
                 ],
                 "VpcId": "vpc-d5c5cab0"
-            },
-            "Metadata": {
-                "AWS::CloudFormation::Designer": {
-                    "id": "29d9f23a-24b8-4297-b96c-18e0465ac54f"
-                }
             }
         },
-        "EC2SG3IL8D": {
+        "LiveStreamLoadBalancerSecurityGroup": {
```

Comment:
good call - we'll be putting [panda](https://github.com/guardian/pan-domain-authentication) in soon to open it up worldwide.